py3status/modules/nvidia_temp.py

```diff
@@ -1,17 +1,23 @@
 # -*- coding: utf-8 -*-
 """
-Display NVIDIA GPU temperature.
+Display NVIDIA GPU temperature(s).
 
 Configuration parameters:
     cache_timeout: how often we refresh this module in seconds (default 10)
-    format_prefix: a prefix for the output. (default 'GPU: ')
-    format_units: the temperature units. Will appear at the end. (default '°C')
-    temp_separator: the separator char between temperatures (only if more than
-        one GPU) (default '|')
+    format: display format for nvidia_temp (default 'GPU: {format_temp}')
+    format_temp: display format for gpu temperature(s) (default '{temp}°C')
+    icon_separator: display separator (only if more than one) (default '|')
+    msg_unavailable: display when unavailable (default 'GPU: N/A')
```

Comment:
@lasers You know what I'm going to say here, but also, this changes the behaviour. the unavailable message has a default of `format_prefix + 'N/A'` which needs fixing anyhow.

Till we have consensus on errors #660 and more.  I'd also not make it available for the user to change